lesson2/Homework_2_6.py

```diff
@@ -0,0 +1,25 @@
+# Нарисовать равнобедренный треугольник из символов ^. Высоту выбирает пользователь.
+
+
+def chek (itr): # проверерим точно ли введено число
+    if itr.isdigit():
+        return 1
+    else:
+        return 0
+
+
+if __name__ == "__main__":
+
+    flg = 0
+    while not flg:
```

Comment:
'while not flg' is bad approach. Remove it everywhere in your files.
```suggestion
    while not number.isdigit():
        number = input()
```